client/mock/api/user/me.ts

```diff
@@ -0,0 +1,23 @@
+import { response } from '../_common/axios_response';
+import type { MeRequestData, MeResponse } from '../models/me_model';
+
+import users from '../../entities/user.json';
+
+/**
+ * Returns a mock response to the - me / whoami / currently authenticated user - request.
+ * @param {Object} data
+ * @returns {AxiosResponse}
+ */
+const mockWhoAmI = (data: MeRequestData): MeResponse => {
+	const userId = getUserIdFromSessionId(data.sessionId);
```

Comment:
Where exactly does sessionId come from?

Reply:
Unfortunately, we will need to pass the JSESSIONID cookie value into the request data for this.
Real Axios can inject it to the header, but I wanted to keep it simple for now. Passing a whole axios request object looks complicated.